twitter-clone/app/src/main/res/layout/activity_main.xml

```diff
@@ -49,10 +49,46 @@
             app:layout_behavior="@string/appbar_scrolling_view_behavior">
 
         <FrameLayout
-                android:id="@+id/fl_main_fragment_layout"
                 android:layout_width="match_parent"
-                android:layout_height="wrap_content" />
-    </android.support.v4.widget.NestedScrollView>
+                android:layout_height="wrap_content">
+
+            <LinearLayout
+                    android:id="@+id/ll_main_trends"
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    android:orientation="vertical"
+                    app:layout_behavior="@string/appbar_scrolling_view_behavior">
```

Comment:
The `layout_behavior` attribute shouldn't be in the parent _FrameLayout_?

Reply:
Actually as the _NestedScrollView_ is the parent of this _FrameLayout_ + _LinearLayout_ you can remove this attribute from the _LinearLayout_.